### PR TITLE
Whohas implement

### PR DESCRIPTION
**Pull request by ghost**
Added The Clear function to make specified player's item being cleared and no harm to server.

### CODE REVIEW

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java [hunk 1]
```diff
@@ -24,8 +24,9 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             return false;
         }
 
-        final boolean doSmite = args.length >= 2 && "smite".equalsIgnoreCase(args[1]);
-
+        final boolean doClear = args.length >= 2 && "clear".equalsIgnoreCase(args[1]);
+		final boolean doSmite = args.length >= 2 && "smite".equalsIgnoreCase(args[1]);
```
Comment (ItzLevvie) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java]: Formatting

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java [hunk 2]
```diff
@@ -47,17 +48,26 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
         {
             if (player.getInventory().contains(material))
             {
                 players.add(player.getName());
-                if (doSmite && !plugin.al.isAdmin(player))
+                if (doClear && !plugin.al.isAdmin(player))
+                {
+                    amount += player.getInventory().clear(material.getId(), -1);
+                }
+				else if (doSmite && !plugin.al.isAdmin(player))
```
Comment (ItzLevvie) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java]: Formatting

**Comment by Wild1145** [comment]
What issue does this resolve?

**Comment by marcocorriero** [comment]
@Wild1145 it adds the clear function , and it makes instead of smiting clearing the specified item, this can avoid mass clearing from admins

**Comment by Wild1145** [comment]
@marcocorriero I know, what issue number does this resolve?

**Comment by marcocorriero** [comment]
#1836

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java [hunk 3]
```diff
@@ -47,17 +48,26 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
...
                 {
                     Command_smite.smite(player);
                 }
             }
         }
+        if(doClear)
```
Comment (ItzLevvie) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_whohas.java]: Needs a space after 'if'

**Comment by Wild1145** [comment]
Formatting needs correcting before this can be merged. Also on hold until associated issue is accepted.

**Comment by ghost** [comment]
@JeromSar and @Wild1145 , Close, @JeromSar added it

**Review by Wild1145 — CHANGES_REQUESTED** [review]
There are some formatting issues that need to be addressed before this could be merged into the project.

**Comment by mayokek** [comment]
@Wild1145, this  feature is already added.

**Comment by lemonsked** [comment]
@marcocorriero, not only they can close. This feature was already added in a previous commit. Closed.

**Pull request closed without merging**